### ISSUE TITLE
Parsing date with time zone specifier "-0000" returns NoneType

### ISSUE THREAD

**Issue by bradley-evans** [scrapinghub/dateparser#548]
I discovered an issue with parsing dates with a time zone specifier of `-0000`. For example:
```
Python 3.7.4 (tags/v3.7.4:e09359112e, Jul  8 2019, 20:34:20) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from dateparser import parse
>>> gooddate = 'Mon, 06 Aug 2018 17:16:32 +0000'
>>> baddate = 'Mon, 6 Aug 2018 18:11:54 -0000'
>>> parse(gooddate)
datetime.datetime(2018, 8, 6, 17, 16, 32, tzinfo=<StaticTzInfo 'UTC\+00:00'>)
>>> parse(baddate)
>>> print(parse(baddate))
None
>>>
```

Version information
```
# pip freeze | findstr dateparser
dateparser==0.7.1
```

**Comment by anarcat** [comment]
i confirm this is a problem that breaks feed2exec with the XKCD feed as seen here:

https://www.xkcd.com/rss.xml

Other offsets and timezones work correctly:

```
In [12]: dateparser.parse(u'Mon, 30 Sep 2019 00:00:00 -0000')

In [13]: dateparser.parse(u'Mon, 30 Sep 2019 00:00:00')
Out[13]: datetime.datetime(2019, 9, 30, 0, 0)

In [14]: dateparser.parse(u'Mon, 30 Sep 2019 00:00:00 UTC')
Out[14]: datetime.datetime(2019, 9, 30, 0, 0, tzinfo=<StaticTzInfo 'UTC'>)

In [15]: dateparser.parse(u'Mon, 30 Sep 2019 00:00:00 -0100')
Out[15]: datetime.datetime(2019, 9, 30, 0, 0, tzinfo=<StaticTzInfo 'UTC\-01:00'>)

```

**Comment by noviluni** [comment]
Hi @bradley-evans  this has been fixed in the last version `0.7.4` :tada:
